arroyo-connectors/src/kafka.rs

```diff
@@ -157,9 +157,23 @@ impl Connector for KafkaConnector {
                         None | Some("latest") => SourceOffset::Latest,
                         Some(other) => bail!("invalid value for source.offset '{}'", other),
                     },
+                    read_mode: match opts.remove("read_mode").as_ref().map(|f| f.as_str()) {
```

Comment:
I think this should be `source.read_mode` and `sink.commit_mode` to match the json and the existing `source.offset` config.

As far as the actual names I'm a bit torn—I do think "read_mode" is a better name than "isolation.level" but there's also benefit in using the existing terminology

Reply:
Going with source.read_mode and sink.commit_mode for now.